.github/workflows/register-validator.yml

```diff
@@ -24,10 +24,6 @@ jobs:
       - name: Check out repository
         uses: actions/checkout@v3
 
-      - name: Setup jq
-        run: |
-          sudo apt install -y jq
```

Comment:
`jq` seems to be used just below : https://github.com/okp4/networks/pull/44/files#diff-78a5b7403401693bbf7258647732c1020cde945975a3162b0090192757544f3dR40 (Line 40) 

```
gentx=$(echo "$GENTX_MD" | tail -n +2 | sed '$d' | jq -c '.')
```

Reply:
Yes. But as far as i see, `jq` is already available by the runner, so there's no need to install it. See:
<img width="666" alt="image" src="https://user-images.githubusercontent.com/9574336/197723129-b6fdbef6-b9ae-4093-8d80-d1075df6af5c.png">

Reply:
Ah yes you're right, sorry :).